src/tracer.js

```diff
@@ -95,10 +99,30 @@ export default class Tracer {
 
         span.addTags(tags);
         span.addTags(internalTags);
+
+        // emit metrics
+        this._metrics.spansStarted.increment(1);
+        if (span.context().isSampled()) {
+            this._metrics.spansSampled.increment(1);
+            if (rpcServer) {
```

Comment:
this is a logical error, if your unit tests are not catching it, they are incomplete.